File: IntegratedHub/ClientApp/src/app/models/feature.ts
export class Feature {

  constructor(Name:string, Description:string, CreatedDate:Date, UpdatedDate:Date, State:boolean) {
    this.Name = Name;
    this.Description = Description;
    this.CreatedDate = CreatedDate;
    this.UpdatedDate = UpdatedDate;
    this.State = State;
  }

  Name: string;
  Description: string;
  CreatedDate: Date;
  UpdatedDate: Date;
  State: boolean;
}

File: IntegratedHub/Models/Feature.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace IntegratedHub.Models
{
    public class Feature
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime UpdatedDate { get; set; }
        public bool State { get; set; }
    }
}
